config.js

```diff
@@ -2,12 +2,12 @@ const configs = {
   production: {
     AUTH_API: 'https://api.andela.com/login?redirect_url=',
     APP_URL: 'https://societies.andela.com',
-    API_BASE_URL: 'https://societies-api.andela.com/api/v1',
+    API_BASE_URL: 'https://api-societies.andela.com/api/v1',
   },
   staging: {
     AUTH_API: 'https://api.andela.com/login?redirect_url=',
-    APP_URL: 'https://societies-staging.andela.com',
-    API_BASE_URL: 'https://societies-api-staging.andela.com/api/v1',
+    APP_URL: 'https://staging-societies.andela.com',
+    API_BASE_URL: 'http://private-ae5c2-andelasocietiesapi.apiary-mock.com/api/v1',
```

Comment:
Why is this using the mock server?

Reply:
@Andretalik Because, our staging is not setup.